docs/make.jl

```diff
@@ -41,7 +41,10 @@ pages = OrderedDict(
             "...improve performance with time series data" => "how_to/improve_ts_performance.md",
             "...write and read data with a JSON" => "how_to/serialize_data.md",
             "...reduce REPL printing" => "how_to/reduce_repl_printing.md",
-            "...migrate from version 3.0 to 4.0" => "how_to/migrating_to_psy4.md"
+            "...handle 3 WT Transformers" => "how_to/handle_3W_transformers.md",
+            "...use PowerSystems.jl with JuMP.jl" => "how_to/jump.md",
```

Comment:
This example is broken with the new cost data in 4.0. We should repair it in this one.